src/validations/access-checker/wearables.ts

```diff
@@ -425,40 +392,61 @@ export const wearables: Validation = {
       return validMerkleProofedEntity
     }
 
+    const parseUrnNoFail = async (urn: string): Promise<DecentralandAssetIdentifier | null> => {
+      try {
+        return await parseUrn(urn)
+      } catch (err) {
+        if (err instanceof Error) {
+          logs?.error(err)
+        } else {
+          logs?.error(`Error parsing urn: ${err}`)
+        }
+      }
+      return null
+    }
+
     const { pointers } = deployment.entity
     const ethAddress = externalCalls.ownerAddress(deployment.auditInfo)
 
-    const resolvedPointers: SupportedAssets[] = []
-    // deduplicate pointer resolution
-    for (const pointer of pointers) {
-      const parsed = await parseUrnNoFail(pointer)
-      if (!parsed)
-        return validationFailed(
-          `Wearable pointers should be a urn, for example (urn:decentraland:{protocol}:collections-v2:{contract(0x[a-fA-F0-9]+)}:{name}). Invalid pointer: (${pointer})`
-        )
-
-      if (!alreadySeen(resolvedPointers, parsed)) resolvedPointers.push(parsed)
+    if (pointers.length == 0) {
+      return validationFailed('Wearable should specify a pointer')
     }
 
-    if (resolvedPointers.length > 1)
-      return validationFailed(`Only one pointer is allowed when you create a Wearable. Received: ${pointers}`)
-
-    const parsed = resolvedPointers[0]
+    const wearablePointer: DecentralandAssetIdentifier | null = await parseUrnNoFail(pointers[0])
+    if (wearablePointer === null) {
+      return validationFailed(`Wearable pointers should be a urn, for example (urn:decentraland:{protocol}:collections-v2:{contract(0x[a-fA-F0-9]+)}:{name}). Invalid pointer: (${pointers[0]})`)
+    }
 
-    if (!validUrnTypes.includes(parsed.type)) {
-      return validationFailed(`Could not resolve the contractAddress of the urn ${parsed}`)
+    const entityTimestampIsAfterADR45 = deployment.entity.timestamp >= ADR_45_TIMESTAMP
+    if (entityTimestampIsAfterADR45) {
```

Comment:
The check should be if `entityTimestampIsAfterADR45 && isL2Wearable`, we must allow deployments of L1 and base wearables with two pointers, even if they are new.

Reply:
Will add